scripts/ci/linux/after_success.sh

```diff
@@ -59,4 +59,24 @@ EOF
 upload_file build/$FILENAME.exe $(sha256sum build/$FILENAME.exe | cut -d' ' -f1) "windows"
 upload_file build/updater.json $(sha256sum build/updater.json | cut -d' ' -f1) "windows"
 
+# Debian package
+echo "Generating changelog for tag ${BUILD_TAG} [${TRAVIS_COMMIT}]"
+
+rm -f debian/changelog
+
+$DOCKER_EXEC "DEBEMAIL=${USER_EMAIL} dch --create --distribution trusty --package \"moolticute\" \
+	--newversion ${DEB_VERSION} \"Release ${BUILD_TAG}\""
+
+echo "Building .deb package..."
+
+$DOCKER_EXEC "cp -f README.md debian/README"
+$DOCKER_EXEC "dpkg-buildpackage -b -us -uc && mkdir -p build-linux/deb && cp ../*.deb build-linux/deb"
+
+# GitHub release
+$DOCKER_EXEC \
+    "TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG} PROJECT_NAME=${PROJECT_NAME} TRAVIS_OS_NAME=${TRAVIS_OS_NAME} \
+    DEB_VERSION=${DEB_VERSION} && \
+    source /usr/local/bin/tools.sh && \
+    create_github_release ${BUILD_TAG}"
```

Comment:
Please avois using `&&` for no reason. `set -e` is already used to stop the script.
it's also not good for maintaining this code, as I was thinking it was only for the docker build, but you do the upload also here....